qt_gui/src/qt_gui/plugin_handler_direct.py

```diff
@@ -122,7 +121,7 @@ def _restore_settings(self, plugin_settings, instance_settings):
                 self._plugin.restore_settings(plugin_settings_plugin, instance_settings_plugin)
             except Exception:
                 qCritical(
-                    'PluginHandlerDirect._restore_settings() plugin "%s" raised an exception:\n%s' %
+                    'PluginHandlerDirect._restore_settings() plugin "%s" raised an exception:\n' %
```

Comment:
This seems wrong. It should still have the second `%s` I'm also quite sure I reviewed this elsewhere. It's a waste of time to be reviewing the same code multiple times. In the future, it would be best to either have a staging branch that you target with pr's for formatting, testing, migration, etc.. and then open a single PR into crystal-devel or to have multiple PR's target crystal-devel (if they don't build or pass the tests you can add an empty file "COLCON_IGNORE" so that you don't break the build). I think the core issue here is that you have 13 commits most of which are likely very small. This many small commits makes it functionally impossible to cherry-pick and drop commits in a clean way. I would highly recommend that you look into using `commit --fixup=` and autosquash to keep the number of commits small and as isolated as possible.

Reply:
Resolved